sources/TileDB.CSharp/Config.cs

```diff
@@ -206,7 +206,7 @@ private sealed class Enumerator(Config config, string prefix) : IEnumerator<KeyV
 
         public bool MoveNext()
         {
-            if (!_iterator.Done())
+            if (_iterator.Done())
```

Comment:
This typo was after I updated the config iterator tests; I will make a patch release after merging this PR.